docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mrcide/poppunk:bacpop-43
+FROM mrcide/poppunk:indiv-viz
```

Comment:
I really wanted to use `mrcide/poppunk:master` here to always get the latest merged poppunk image but it doesn't seem to exist! This is the latest on docker hub.